sycl/source/detail/memory_manager.cpp

```diff
@@ -165,9 +165,8 @@ void copyH2D(SYCLMemObjT *SYCLMemObj, char *SrcMem, QueueImplPtr SrcQueue,
              unsigned int DimDst, sycl::range<3> DstSize,
              sycl::range<3> DstAccessRange, sycl::id<3> DstOffset,
              unsigned int DstElemSize, std::vector<cl_event> DepEvents,
-             bool UseExclusiveQueue, cl_event &OutEvent) {
```

Comment:
Also even if we do not use UseExclusiveQueue anymore inside of this function, I'd rather we keep it so the interface remains the same for now...